stylesheet.css

```diff
@@ -373,3 +394,493 @@ a {
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    margin-top: 10px;
+    margin-bottom: 60px;
+  }
+
+  .display-project1 {
+    flex-direction: row;
+  }
+
+  .display-project2 {
+    flex-direction: row-reverse;
+  }
+
+  .display-project3 {
+    flex-direction: row;
+  }
+
+  .display-project4 {
+    flex-direction: row-reverse;
+  }
+
+  .project-flex {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    padding-left: 23px;
+  }
+
+  .flex-item {
+    color: white;
+    align-items: center;
+    width: 100%;
+  }
+
+  .holdtext {
+    width: 100%;
+    color: #dfe1e6;
+    font-size: 20px;
+  }
+
+  .navlang-flex {
+    display: flex;
+    flex-wrap: nowrap;
+    width: 100%;
+  }
+
+  .subflex-item {
+    height: 30px;
+    text-align: center;
+    line-height: 30px;
+  }
+
+  .project-text-nav {
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    width: 40%;
+    height: 100%;
+    margin-top: 20px;
+  }
+
+  .hh2 {
+    width: 327px;
+    font-size: 30px;
+    font-weight: 500;
+    text-align: left;
+  }
+
+  .placeholder-div {
+    width: 59%;
+    height: 448px;
+    padding-bottom: 10px;
+  }
+
+  .ph-image {
+    width: 95%;
+    height: 450px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: grid;
+  }
+
+  .navlang-flex li {
+    width: auto;
+    display: inline-block;
+  }
+
+  .seeproject {
+    display: inline-block;
+    color: #edecff;
+    width: 89px;
+    height: 50px;
+    background-color: #7d73fc;
+    margin-top: 80px;
+    line-height: 48px;
+    text-align: center;
+  }
+
+  .seep {
+    display: flex;
+    justify-content: left;
+    height: 10px;
+    width: 100%;
+    padding-bottom: 10px;
+  }
+
+  .contact-section,
+  .foota {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    width: 80%;
+    margin: auto;
+  }
+
+  .contact-section {
+    margin-top: 80px;
+  }
+
+  .cc-sect {
+    width: 100%;
+  }
+
+  .cc-top {
+    padding-top: 40px;
+  }
+
+  .contact {
+    display: block;
+    width: 60%;
+    color: #9390ff;
+    font-size: 32px;
+    text-align: center;
+    padding-bottom: 30px;
+    margin: auto;
+  }
+
+  .mble {
+    display: none;
+  }
+
+  .text-invite {
+    width: 60%;
+    height: 105px;
+    font-size: 20px;
+    color: #ebebff;
+    text-align: center;
+    margin: auto;
+  }
+
+  .myform {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    width: 100%;
+  }
+
+  .myform input {
+    width: 60%;
+    height: 48px;
+    margin-bottom: 15px;
+  }
+
+  .myform textarea {
+    width: 60%;
+    height: 148px;
+    margin-bottom: 15px;
+  }
+
+  .myform .dsk {
+    display: block;
+    width: 100px;
+    height: 50px;
+    background-color: #7d73fc;
+    border-style: none;
+    transition: width 3s, backgeround-color 4s;
+  }
```

Comment:
- [ ] kindly fix the style of the contact button to match Figma

- [ ] [OPTIONAL] I would suggest that you add a `cursor: pointer;` style to your  button as well to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

Figma            |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/187971187-a4328656-24ff-46fc-8042-600e2b6d78a0.png)  |  ![image](https://user-images.githubusercontent.com/89282221/187971247-848f57ab-0fad-40fc-8feb-8591b717d4d5.png)